app.js

```diff
@@ -0,0 +1,28 @@
+const http = require("http");
+
+require("dotenv").config();
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+app = express();
+app.use(express.json());
+app.use(cors());
+app.use(morgan("tiny"));
+
+app.get("/ping", (req, res) => {
+  res.status(200).json({ message: "pong" });
+});
+
```

Comment:
```suggestion

require("dotenv").config();

const express = require("express");
const cors = require("cors");
const morgan = require("morgan");

const app = express();

app.use(express.json());
app.use(cors());
app.use(morgan("tiny"));

app.get("/ping", (req, res) => {
  res.status(200).json({ message: "pong" });
});

```
가독성을 위해 줄간격을 띄워주세요!
app을 선언할 때는 const 를 사용하지 않은 이유가 있을까요?